### DISCUSSION TITLE
Is the PyGithub project dead? How can the community help?

### DISCUSSION THREAD

**Discussion by EnricoMi** [PyGithub/PyGithub#2205] [General]
Over the last months, I have seen many very good pull requests from the community that fix blocking bugs or add new features, as well as many questions around issues using this library. This project is actively being used by [25k (public) Github repositories](https://github.com/PyGithub/PyGithub/network/dependents?package_id=UGFja2FnZS01MjQ4NDg2OQ%3D%3D).

Unfortunately, this projects seems to be under-equipped with active maintainers. While the [MAINTAINERS](https://github.com/PyGithub/PyGithub/blob/master/MAINTAINERS) file lists six maintainers, there is only one lonely soul left driving this project. Blocking issues are mounting, ready pull requests are not reviewed, approved or merged, users getting annoyed by the lack of response while Github API is actively developing away from this Python implementation.

This is not sustainable.

Everyone knows and appreciates that this is open-source software and maintainers are putting their free time into this project. But this is no excuse to not improve the situation.

How can the community help? What we need the maintainer to do? See issue #2178.

**Comment by EnricoMi** [comment]
This project is so actively been used, so many questions, issues and open pull requests. But no attempt is done by the maintainers to open it up to the community.

[MAINTAINERS](https://github.com/PyGithub/PyGithub/blob/master/MAINTAINERS), please mark this project as dead so the community is not wasting more time here.

Community, feel free to fork this repository, get all the good work merged and released there. Post your maintained fork here.

**Comment by dawngerpony** [comment]
Hi @EnricoMi, is this thread still valid? How do you feel about the status of the project these days?

**Comment by EnricoMi** [comment]
Thanks for asking, I think this answer your question best: https://github.com/PyGithub/PyGithub/issues/2178#issuecomment-1952083571
